### PR TITLE
[4.5] Fix CA/server cert validation in FIPS

### PR DESCRIPTION
**Pull request by stlaz**
In FIPS, the NSS library needs to be passed passwords to perform
certificate validation. Should we not have passed it and the NSS
guys have not fixed this yet, we would get SEC_ERROR_BAD_SIGNATURE
which is completely different error than one would expect but
that's just how things are with NSS right now.

https://pagure.io/freeipa/issue/6897

### CODE REVIEW

File: ipapython/certdb.py [hunk 1]
```diff
@@ -77,6 +77,12 @@ def find_cert_from_txt(cert, start=0):
     return (cert, e)
 
 
+def get_file_cont(slot, token, filename):
+    with open(filename) as f:
```
Comment (Akasurde) [inline ipapython/certdb.py]: LGTM

File: ipapython/certdb.py [hunk 2]
```diff
@@ -77,6 +77,12 @@ def find_cert_from_txt(cert, start=0):
...
+        cont = f.read()
```
Comment (tiran) [inline ipapython/certdb.py]: ```return f.read()```

**Comment by stlaz** [comment]
Made a quickfix according to @tiran, the ACK can stay. Thanks, I was being paranoid.

**Comment by MartinBasti** [comment]
ipa-4-5:

* 651d132b701b773b2bbeb41496d6c5ddbf6d19b3 Fix CA/server cert validation in FIPS

**Pull request closed without merging**